misc/git-hooks/pre-commit

```diff
@@ -92,7 +92,7 @@ function check_check_forbidden_dependencies() {
       return 1
     fi
     find modules/ -name Cargo.toml | grep common/ | while read -r cargo_toml ; do
-      if grep -E "fedimint-" "$cargo_toml" | grep -E -v "fedimint-core|fedimint-api-client|-common|fedimint-logging" >&2 ; then
+      if grep -E "fedimint-" "$cargo_toml" | grep -E -v "fedimint-core|-common|fedimint-logging|fedimint-connectors" >&2 ; then
```

Comment:
I always forget how to properly modify the git hooks. @dpc am I supposed to modify this directly or is there a command to get flakebox to do it?

Reply:
Yes, this file is auto-generated (see top of the file header).

@m1sterc001guy 

```
> rg "find modules"
misc/git-hooks/pre-commit
94:    find modules/ -name Cargo.toml | grep common/ | while read -r cargo_toml ; do

nix/check-forbidden-deps.sh
13:find modules/ -name Cargo.toml | grep common/ | while read -r cargo_toml ; do
>
```

`nix/check-forbidden-deps.sh` is the source, then reload dev shell and flakebox will offer to re-generate.